test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
```

Comment:
Let's say that I have the situation
```typescript
const foo = bar;
```
The `const` in the above only affects the usage of `foo`.  It does not care about what `bar` is (even if `bar` is mutable or `immutable`)

How does it affect the usage of `foo`?  Mainly, that `foo` cannot be switched to some different thing.  This makes it so that
```typescript
foo = baz;
```
is disallowed.  The switching portion is key here.